src/components/SignUp/index.jsx

```diff
@@ -1,13 +1,67 @@
 import React from "react";
 import { Container, Row, Col } from "react-bootstrap";
+import { Link, useHistory } from "react-router-dom";
 import SignUpForm from "../SignUpForm";
 import womenSupportEachOther from "../../images/womenSupportEachOther.png";
 import googleIcon from "../../images/icons/googleIcon.svg";
 import twitterIcon from "../../images/icons/twitterIcon.svg";
 import facebookIcon from "../../images/icons/facebookIcon.svg";
+import firebase from "../../firebaseConfig";
+import { AuthContext } from "../../auth/Authentication";
 import "./index.scss";
 
 export default function SignUp() {
+  const history = useHistory();
+
+  const signUpSucceed = () => {
+    history.push("/");
+  };
+
+  const signUpFailed = (error) => {
+    alert(error);
```

Comment:
is it just alerting? will you add modal or something later?

Reply:
I added a new validation error for the case that we want to handle, other error should be ignored of just pop up a modal to say the an error occurs, try again later. If we have time, we can add that modal but till now, that's enought